pyiron_base/state/settings.py

```diff
@@ -333,7 +333,7 @@ def _get_config_from_file(self) -> Union[Dict, None]:
             config_file = os.path.expanduser(os.path.join("~", ".pyiron"))
 
         if os.path.isfile(config_file):
-            parser = ConfigParser(inline_comment_prefixes=(";",))
+            parser = RawConfigParser(inline_comment_prefixes=(";",))
```

Comment:
Is the issue that arbitrary passwords can trigger the interpolation by accident or does `ConfigParser` generally refuse "weird" characters for option values?  If it's only the former I would be more comfortable with
 ```suggestion
            parser = ConfigParser(inline_comment_prefixes=(";",), interpolation=None)
```
since `RawConfigParser` is marked as legacy in the docs.